src/time-utils.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+const ms = require('ms')
+
+function staleDaysToDate(input) {
+  try {
+    const staleDays = Number(input)
+    if (isNaN(staleDays)) {
+      const stringToMs = ms(input)
+      return new Date().getTime() - stringToMs
+    }
+    return new Date().getTime() - daysToMs(staleDays)
+  } catch (error) {
+    return new Date().getTime()
+  }
+}
+
+function isCommitStale(unreleasedCommits, staleDate) {
+  if (!unreleasedCommits || !unreleasedCommits.length) return false
```

Comment:
remove this check. when a function receives an input, it's responsibility of the caller to provide proper input. also rename as per previous comment